VirtualBox guest utils depend on current Linux kernel, but since the installed kernel might not be
up-to-date, it is best that we prevent the guest utils from being upgraded during system upgrades.

--- /etc/pacman.conf
+++ /etc/pacman.conf
@@ -22,7 +22,7 @@
 Architecture = auto

 # Pacman won't upgrade packages listed in IgnorePkg and members of IgnoreGroup
-#IgnorePkg   =
+IgnorePkg    = linux linux-firmware linux-headers virtualbox-guest-utils-nox virtualbox-guest-modules
 #IgnoreGroup =

 #NoUpgrade   =
